_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -838,6 +835,27 @@ public function can_request( $request ) {
 			return current_user_can( 'jetpack_configure_modules' );
 		}
 	}
+
+	/**
+	 * Allows the `settings` and `module/<slug>` EDITABLE endpoints to accept both JSON and multi-part POST bodies.
+	 *
+	 * @param WP_REST_Request $request The request sent to the WP REST API.
+	 *
+	 * @return array|bool
+	 */
+	public function parse_settings_request_body( $request ) {
```

Comment:
James, with this function I was trying to avoid calling `get_json_params` for any settings-related request, which is how we are currently doing it. See: https://github.com/Automattic/jetpack/pull/6006/files#diff-2ccec9069213650359171727de9a2471L426

Calling `get_json_params` only allows for JSON encoded POST bodies. This function allows for both.